src/DB_Command.php

```diff
@@ -789,120 +795,124 @@ public function size( $args, $assoc_args ) {
 		}
 
 		// Build rows for the formatter.
-		$rows = array();
+		$rows   = array();
 		$fields = array( 'Name', 'Size' );
 
 		$default_unit = ( empty( $size_format ) && ! $human_readable ) ? ' B' : '';
 
 		if ( $tables || $all_tables || $all_tables_with_prefix ) {
 
 			// Add all of the table sizes
-			foreach( WP_CLI\Utils\wp_get_table_names( $args, $assoc_args ) as $table_name ) {
+			foreach ( WP_CLI\Utils\wp_get_table_names( $args, $assoc_args ) as $table_name ) {
 
 				// Get the table size.
-				$table_bytes = $wpdb->get_var( $wpdb->prepare(
-					"SELECT SUM(data_length + index_length) FROM information_schema.TABLES where table_schema = '%s' and Table_Name = '%s' GROUP BY Table_Name LIMIT 1",
-					DB_NAME,
-					$table_name
+				$table_bytes = $wpdb->get_var(
+					$wpdb->prepare(
+						'SELECT SUM(data_length + index_length) FROM information_schema.TABLES where table_schema = %s and Table_Name = %s GROUP BY Table_Name LIMIT 1',
+						DB_NAME,
+						$table_name
 					)
 				);
 
 				// Add the table size to the list.
 				$rows[] = array(
-					'Name'  => $table_name,
-					'Size'  => strtoupper( $table_bytes ) . $default_unit,
+					'Name' => $table_name,
+					'Size' => strtoupper( $table_bytes ) . $default_unit,
 				);
 			}
 		} else {
 
 			// Get the database size.
-			$db_bytes = $wpdb->get_var( $wpdb->prepare(
-				"SELECT SUM(data_length + index_length) FROM information_schema.TABLES where table_schema = '%s' GROUP BY table_schema;",
-				DB_NAME
+			$db_bytes = $wpdb->get_var(
+				$wpdb->prepare(
+					'SELECT SUM(data_length + index_length) FROM information_schema.TABLES where table_schema = %s GROUP BY table_schema;',
+					DB_NAME
 				)
 			);
 
 			// Add the database size to the list.
 			$rows[] = array(
-				'Name'  => DB_NAME,
-				'Size'  => strtoupper( $db_bytes ) . $default_unit,
-				);
+				'Name' => DB_NAME,
+				'Size' => strtoupper( $db_bytes ) . $default_unit,
+			);
 		}
 
 		if ( ! empty( $size_format ) || $human_readable ) {
-			foreach( $rows as $index => $row ) {
+			foreach ( $rows as $index => $row ) {
 					// These added WP 4.4.0.
```

Comment:
Comment still badly indented here, but it also does not make any sense. Let's remove it.
```suggestion
```